datomic.py

```diff
@@ -23,19 +30,29 @@ def __init__(self, location, storage):
     def db_url(self, dbname):
         return urljoin(self.location, 'data/') + self.storage + '/' + dbname
 
-    def create_database(self, dbname):
+    def create_database(self, dbname, stream=False, size=None):
         r = requests.post(self.db_url(''), data={'db-name':dbname})
+        if gevent:
```

Comment:
Instead of doing this, you probably want to abstract out the transport layer from the database, and pass the transport in Database.__init__().